tasks/configure.yml

```diff
@@ -2,10 +2,11 @@
 - name: Ensure ccache cache directory exists
   become: true
   file:
-    path: "{{ ccache_cache_dir }}"
```

Comment:
Does this mean we can remove `ccache_cache_dir`?

Reply:
Do you mean from our other projects which use this role? If so, yes. If you mean from this role itself, then b926b35 effectively removes this var.

Reply:
Got it, so previously this var had to be set by the roles which depend on it, but now it's automatically inferred.

Reply:
`ccache_cache_dir` was not documented as being required (on the `README.md` or elsewhere), which means that this commit is technically a bugfix. Please reword the commit message to make it clear.